src/systems/systems/src/hydraulic/mod.rs

```diff
@@ -231,14 +266,16 @@ impl PowerTransferUnit {
 
     const DELAY_TO_DECLARE_CONTINUOUS: Duration = Duration::from_millis(1500);
     const THRESHOLD_DELTA_TO_DECLARE_CONTINUOUS_RPM: f64 = 400.;
+    const DURATION_BEFORE_CAPTURING_BARK_STRENGTH_SPEED: Duration = Duration::from_millis(133);
 
     pub fn new(context: &mut InitContext) -> Self {
+        let randomized_is_ptu_worn_out = Self::randomized_is_ptu_worn_out();
         Self {
-            active_l2r_id: context.get_identifier("HYD_PTU_ACTIVE_L2R".to_owned()),
-            active_r2l_id: context.get_identifier("HYD_PTU_ACTIVE_R2L".to_owned()),
-            motor_flow_id: context.get_identifier("HYD_PTU_MOTOR_FLOW".to_owned()),
             valve_opened_id: context.get_identifier("HYD_PTU_VALVE_OPENED".to_owned()),
             shaft_rpm_id: context.get_identifier("HYD_PTU_SHAFT_RPM".to_owned()),
+            dev_delta_pressure: context.get_identifier("HYD_PTU_DEV_DEACTIVATION_DELTA".to_owned()),
+            bark_strength_id: context.get_identifier("HYD_PTU_BARK_STRENGTH".to_owned()),
+            dev_efficiency_id: context.get_identifier("HYD_PTU_DEV_EFFICIENCY".to_owned()),
```

Comment:
Yeah I have mixed feelings because they are super nice to use for some purposes like tests and making a showcase of one particular situation and so on, but on the other hand I don't want anyone to just input random values in there or use them for production purpose....